pipe/section/section_impls/mysql_connector/src/destination.rs

```diff
@@ -140,13 +140,14 @@ impl Mysql {
 // FIXME: move this function to lib
 fn to_naive_date(tu: TimeUnit, t: i64) -> Option<NaiveDateTime> {
     match tu {
-        TimeUnit::Second => NaiveDateTime::from_timestamp_opt(t, 0),
-        TimeUnit::Millisecond => NaiveDateTime::from_timestamp_micros(t * 1000),
-        TimeUnit::Microsecond => NaiveDateTime::from_timestamp_micros(t),
+        TimeUnit::Second => DateTime::from_timestamp(t, 0),
+        TimeUnit::Millisecond => DateTime::from_timestamp_micros(t * 1000),
+        TimeUnit::Microsecond => DateTime::from_timestamp_micros(t),
         TimeUnit::Nanosecond => {
-            NaiveDateTime::from_timestamp_opt(t / 1_000_000_000, (t % 1_000_000_000) as _)
+            DateTime::from_timestamp(t / 1_000_000_000, (t % 1_000_000_000) as _)
         }
     }
+    .map(|d| d.naive_utc())
```

Comment:
This seems like an extra step, why is it necessary?

Reply:
chrono deprecated previous methods, linter yelling loudly in gh actions